src/toil/test/wdl/builtinTest.py

```diff
@@ -251,6 +251,13 @@ def test_sub(self):
         # preserving the path. Cromwell would return 'src/toil/test/wdl/test.csv' instead.
         self.check_function('sub', cases=['as_input_with_file'], expected_result='test.csv')
 
+    def test_size(self):
+        self.check_function('size', cases=['as_command'], expected_result='19.0')
+
+        # the output file contains 22 bytes in bytes, K, and Ki separated with spaces.
+        # NOTE: Cromwell returns '22.0 0.021484375 0.021484375' where the binary version is returned for K.
```

Comment:
This comment is difficult to understand. Can context be added for newbies (or people that haven't looked at the code for a long time)?